WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -497,16 +481,13 @@ public class ReaderDetailViewController : UIViewController, UIViewControllerRest
 
     }
 
+
     private func configureRichText() {
-        richTextView.delegate = self
-        let fontSize = WPStyleGuide.Detail.contentFontSize
-        let lineHeight = WPStyleGuide.Detail.contentLineHeight
-        var textOptions = richTextView.textOptions
-        textOptions[DTDefaultFontSize] = WPStyleGuide.Cards.contentFontSize
-        textOptions[DTDefaultLineHeightMultiplier] = lineHeight / fontSize
```

Comment:
@koke i hope you **do** realize.. that gif of yours... is a game changer to me. Right?